### ISSUE TITLE
[Feature] Allow ['camera', 'microphone'] permissions preset for WebKit browser

### ISSUE THREAD

**Issue by Xotabu4** [microsoft/playwright#28430]
Currently when trying to set ['camera', 'microphone'] permissions for webkit browser, error is thrown `Error: browserContext.newPage: Unknown permission: microphone`

But when working in Webkit manually - it seems like everything to support fake webcam is there, only pre-setting permissions for webkit should be implemented.

Would be really valuable for webRTC testing to have ability to run webkit with fake webcam and microphone.

![image](https://github.com/microsoft/playwright/assets/3033972/d0e64d78-ac1e-4b38-bb80-ff9e3b10fbeb)
![image](https://github.com/microsoft/playwright/assets/3033972/ce133728-e8a9-4c89-a7d2-dbc2438378fa)
![image](https://github.com/microsoft/playwright/assets/3033972/a71f1b27-2ee9-43d1-8f81-8e3ec0e5c635)

**Comment by acanyasar** [comment]
I am testing a video communication app. This would be very helpful. @pavelfeldman

**Comment by giovegaq** [comment]
I'm getting this issue

**Comment by zayehalo** [comment]
This is also crucial for us to be able to test our apps on webkit.

**Comment by erickitomaz** [comment]
Video sessions is one of our chore features in our app, would be nice to have this feature already implemented.

**Comment by DvaKompota** [comment]
Needed for testing bar/QR codes with iPads

**Comment by denkorz** [comment]
This issue looks like a blocker for using playwright

**Comment by Xotabu4** [comment]
@denkorz yes, unfortunately we had to limit our testing to chrome browser for now due to this issue.

**Comment by giovegaq** [comment]
As a workaround I click on the "allow" button using an "Automator" script

exec(`osascript ${path.resolve("./scripts/clickAllowButton.scpt")}`);      

It works really well while executing a Playwright script on headless and Webkit, the problem is that only works on Mac.

this is my clickAllowButton.scpt script (if it doesn´t work you can create your own script): 

on is_running()
	tell application "System Events" to (name of processes) contains "Playwright"
end is_running

set safRunning to is_running()

if safRunning then
	tell application "Automator" to activate
	delay 2
	tell application "System Events"
		try
			click UI element "Allow" of sheet 1 of window 1 of application process "Playwright"
		end try
	end tell
else
	return "Playwright is not running"
end if

**Comment by YauheniPo** [comment]
> exec(`osascript ${path.resolve("./scripts/clickAllowButton.scpt")}`);

It works for my tests. thanks
